tmt.spec

```diff
@@ -121,6 +121,8 @@ export SETUPTOOLS_SCM_PRETEND_VERSION=%{version}
 %install
 %pyproject_install
 %pyproject_save_files tmt
+# Executable only used for pre-commit. Should not be installed
+rm %{buildroot}/%{_bindir}/tmt-pre-commit
```

Comment:
I don't think we should have discrepancies like this between rpms and wheels.  
Reading the previous comments, it seems dedicated command is preferable, though I'd like to propose having another look into having this as a hidden argument instead.

Reply:
Thanks for the nice write up @LecrisUT!  

I don't know what's the best way to do the actual invocation, my point is that we shouldn't delete anything from the rpm, that is shipped with the wheel. Or rather, not add the entry point in `project.scripts` if it's not also shipped in rpm. Can we do `python -m` or something instead?

Reply:
> Can we do python -m or something instead?

I think it should work, let me push that change and see if it works

Reply:
Nice, thanks! Do we need `/packit build` and/or `/packit test`? How does it work these days? :)

Reply:
Yes, `/packit build` and then `/packit test`